frontend/src/modules/member/member-service.js

```diff
@@ -127,6 +127,44 @@ export class MemberService {
     return response.data
   }
 
+  static async listActive({
+    platform,
+    isTeamMember,
+    activityTimestampFrom,
+    activityTimestampTo,
+    orderBy,
+    offset,
+    limit
+  }) {
+    const params = {
+      ...(platform.length && {
+        'filter[platforms]': platform
+          .map((p) => p.value)
+          .join(',')
+      }),
+      ...(isTeamMember === false && {
+        'filter[isTeamMember]': isTeamMember
```

Comment:
We might need to add the `filter[isBot]: isBot` here as well after merging the #465

Reply:
Thanks for the heads up! I'll wait for your PR to update this branch